lib/oban/midwife.ex

```diff
@@ -21,6 +21,96 @@ defmodule Oban.Midwife do
     GenServer.start_link(__MODULE__, opts, name: name)
   end
 
+  @doc """
```

Comment:
This shouldn't be documented publicly. The public interface for this is _always_ through `Oban.start_queue`.

Reply:
This `@doc` will not be processed by ex_doc given that the `@moduledoc` is `false`, I added this mainly as a reference for other developers.

Reply:
Understood. I don't want to maintain the same docs and options in both places though. We could put in a `@doc see Oban.start_queue/2` or something (I think there is a syntax for doing that cleanly, but I don't recall it).